Dshop/apps/products_catalogue/tests/test_persistent_cart.py

```diff
@@ -76,3 +76,20 @@ def test_add_non_exist_product_to_cart(client):
     fake_response = client.post(url)
 
     assert fake_response.status_code == 404
+
+
+@mark.dj_shop_cart
+@pytest.mark.django_db
+def test_response_add_to_cart_zero_view_request(client, tv_product):
```

Comment:
This test makes kind of zero sense. 
It only tests redirection, not if there was any change to the cart. 
Can you add some test to show that the cart did not change?

Reply:
Or check if message was really added.

Reply:
Yes. You're absolutely right. I made it too fast.